bot/exts/evergreen/error_handler.py

```diff
@@ -128,6 +133,35 @@ async def on_command_error(self, ctx: commands.Context, error: commands.CommandE
 
             log.exception(f"Unhandled command error: {str(error)}", exc_info=error)
 
+    async def send_command_suggestion(self, ctx: commands.Context, command_name: str) -> None:
+        """Sends user similar commands if any can be found."""
+        raw_commands = []
+        for cmd in self.bot.walk_commands():
+            if not cmd.hidden:
+                raw_commands += (cmd.name, *cmd.aliases)
+        if similar_command_data := difflib.get_close_matches(command_name, raw_commands, 1):
+            similar_command_name = similar_command_data[0]
+            similar_command = self.bot.get_command(similar_command_name)
+
+            if not similar_command:
+                return
+
+            log_msg = "Cancelling attempt to suggest a command due to failed checks."
+            try:
+                if not await similar_command.can_run(ctx):
+                    log.debug(log_msg)
+                    return
+            except commands.errors.CommandError as cmd_error:
+                log.debug(log_msg)
+                await self.on_command_error(ctx, cmd_error)
+                return
+
+            misspelled_content = ctx.message.content
+            e = Embed()
+            e.set_author(name="Did you mean:", icon_url=QUESTION_MARK_ICON)
+            e.description = f"{misspelled_content.replace(command_name, similar_command_name, 1)}"
```

Comment:
`misspelled_content` ([`ctx.message.content`](https://discordpy.readthedocs.io/en/latest/api.html#discord.Message.content)) is already a string. No need to convert it into one again.
```suggestion
            e.description = misspelled_content.replace(command_name, similar_command_name, 1)
```
I'm guessing `misspelled_content` can't be directly replaced by `ctx.message.content` because it would go over the column limit?

Reply:
> I'm guessing misspelled_content can't be directly replaced by ctx.message.content because it would go over the column limit?

Yeah.